pkg/crd/markers/crd.go

```diff
@@ -55,6 +55,9 @@ var CRDMarkers = []*definitionWithHelp{
 
 	must(markers.MakeDefinition("kubebuilder:metadata", markers.DescribesType, Metadata{})).
 		WithHelp(Metadata{}.Help()),
+
+	must(markers.MakeDefinition("kubebuilder:selectablefield", markers.DescribesType, SelectableField{})).
```

Comment:
Let's add a few of these to https://github.com/kubernetes-sigs/controller-tools/blob/7ef21aae5452047af82f020774639c94aa659eca/pkg/crd/testdata/cronjob_types.go#L636

IIRC they then also have to be added here: https://github.com/kubernetes-sigs/controller-tools/blob/7ef21aae5452047af82f020774639c94aa659eca/pkg/crd/testdata/testdata.kubebuilder.io_cronjobs.yaml

(but you can just check the failing test after the marker was added to the type)